documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc

```diff
@@ -21,22 +21,7 @@ endif::[]
 [id="about-mtv"]
 == About {the-lc} {project-full}
 
-You can migrate virtual machines from VMware vSphere, {rhv-full}, or {osp} source providers to {virt} with {the-lc} {project-first}.
-
-Migration using OpenStack source providers is a Technology Preview.
-
-[IMPORTANT]
-====
-Migration using OpenStack source providers is a Technology Preview feature only. Technology Preview features
-are not supported with Red Hat production service level agreements (SLAs) and
-might not be functionally complete. Red Hat does not recommend using them
-in production. These features provide early access to upcoming product
-features, enabling customers to test functionality and provide feedback during
-the development process.
-
-For more information about the support scope of Red Hat Technology Preview
-features, see https://access.redhat.com/support/offerings/techpreview/.
-====
+You can migrate virtual machines from VMware vSphere, Open Virtual Application (OVA) files that were created by or are compatible with VMware vSphere, {rhv-full}, {ocp-short}, or {osp} source providers to {virt} with {the-lc} {project-first}.
```

Comment:
```suggestion
You can migrate virtual machines from VMware vSphere, Open Virtual Appliances (OVA) that were created by VMware vSphere, {rhv-full}, {osp}, or {virt} source providers to {virt} with {the-lc} {project-first}.
```
Application -> Appliance
OVA files -> OVA (since OVA might be a folder)
and I'd place OCP-to-OCP as the last one and make it specific to OpenShift Virtualization